decidim-core/lib/decidim/core/test/shared_examples/embed_resource_examples.rb

```diff
@@ -11,22 +29,25 @@
     include_context "with a component"
   end
 
-  context "when visiting the embed page for a resource" do
-    before do
-      visit resource_locator(resource).path
-      visit "#{current_path}/embed"
-    end
+  unless options.is_a?(Hash) && options[:skip_unpublish_checks]
```

Comment:
I feel this is a little bit backwards as a check. Maybe rename the option to something like `skip_publication_checks`? 

```suggestion
  unless options.is_a?(Hash) && options[:skip_unpublish_checks]
```

Reply:
👍🏽, the current naming is bad